tools/common/config/validation.go

```diff
@@ -59,3 +59,18 @@ func ValidateIPv4LoopbackCidr(cidr string) error {
 	}
 	return nil
 }
+
+// copied over from the dependency implementation package to avoid circular imports
+const (
+	legacy = "legacy"
+	nft    = "nft"
+)
+
+func ValidateIptablesVersion(iptv string) error {
+	switch iptv {
+	case "", legacy, nft:
+		return nil
+	default:
+		return fmt.Errorf("iptables version %s not supported", iptv)
```

Comment:
```suggestion
		return fmt.Errorf("Invalid FORCE_IPTABLES_BINARY value %q", iptv)
```

Reply:
fixed in https://github.com/istio/istio/pull/57976/commits/a97ba6839d1c1c2a16aa4c0cd085f6854fdfc244